.github/workflows/build-test.yml

```diff
@@ -69,10 +69,11 @@ jobs:
                 PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
               run: /usr/local/bin/bash ./ci/test_mac.sh
-            - uses: actions/upload-artifact@master
+            - name: Upload lcov coverage directory
+              uses: actions/upload-artifact@master
```

Comment:
maybe here  you can use concrete version of upload-artifact instead of master

Reply:
set to v1